src/index.js

```diff
@@ -0,0 +1,41 @@
+import './style.css';
+
```

Comment:
- [ ] You are meant to have one new JavaScript file imported as a module containing methods related to the status updates (completed: true / false). I there fore kindly suggest creating a new `.js` file and utilize the import feature 😄